docs/index.mdx

```diff
@@ -84,13 +84,13 @@ Before you are able to display ads to your users, you must have a [Google AdMob
 "Apps" menu item, create or choose an existing Android/iOS app. Each app platform exposes a unique account ID which needs to
 be added to the project.
 
-> Attempting to build your app without a valid App ID in `app.json` or `app.config.js` will cause the app to crash on start or fail to build.
+> Attempting to build your app without a valid App ID in `app.json` or `app.config.js/ts` will cause the app to crash on start or fail to build.
```

Comment:
here we are in backticks so I think we can use regex style to make it clear this is not a directory with a `ts` entry

```suggestion
> Attempting to build your app without a valid App ID in `app.json` or `app.config[.js|.ts]` will cause the app to crash on start or fail to build.
```